htdocs/inc.header.php

```diff
@@ -294,6 +296,7 @@ function fileGetContentOrDefault($filename, $defaultValue)
     'volume' => "/usr/bin/sudo ".$conf['scripts_abs']."/playout_controls.sh -c=setvolume -v=%s",            // change volume
     'maxvolume' => "/usr/bin/sudo ".$conf['scripts_abs']."/playout_controls.sh -c=setmaxvolume -v=%s",      // change max volume
     'startupvolume' => "/usr/bin/sudo ".$conf['scripts_abs']."/playout_controls.sh -c=setstartupvolume -v=%s",      // change startup volume
+    'bootvolume' => "/usr/bin/sudo ".$conf['scripts_abs']."/playout_controls.sh -c=setbootvolume -v=%s",      // change startup volume
```

Comment:
Remark: The comment at the should also be bootvolume

Reply:
@s-martin what do you mean?

Reply:
forget it, I know what you meant :)